templates/core/index.html

```diff
@@ -6,18 +6,63 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1QoM0Yqx4JfDJ2G3OPt5bds-ximG1jASmU_PWX4iff3E/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip--image p-banner-core-hero is-deep is-dark">
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Build secure IoT devices with Ubuntu Core</h1>
-      <!-- small screen version -->
-      <img class="u-visible--small u-hide--medium u-hide--large" src="{{ ASSET_SERVER_URL }}7b563def-core_white-orange_st_hex.svg" width="150" alt="" style="margin: 4rem 0;"/>
-      <h4>Everything you love about Ubuntu, locked down for security. Helping you make safer things - because we’re all connected.</h4>
-    </div>
-    <div class="col-5 u-align--center u-hide--small u-vertically-center">
-      <!-- medium and large screen version -->
-      <img src="{{ ASSET_SERVER_URL }}7b563def-core_white-orange_st_hex.svg" width="156" alt="" />
+<section class="p-takeover">
+  <div class="p-strip--image is-dark is-deep p-takeover__suru">
+    <div class="row u-equal-height">
+      <div class="col-8">
+        <h1>An introduction to IoT app stores</h1>
+        <p class="p-heading--four">Curate your own customised store for a fully managed device software portfolio</p>
```

Comment:
All of our other takeover subheadings have a full stop at the end, we probably want one here too.